presto-docs/src/main/sphinx/sql/explain.rst

```diff
@@ -152,6 +152,33 @@ IO:
        }
      }
 
+To include warnings in the EXPLAIN (TYPE IO) output, set the configuration property ``print-warnings-for-explain-io`` or session property ``print_warnings_for_explain_io`` to true
+
+.. code-block:: none
```

Comment:
```suggestion
.. code-block:: none

```
Not having a blank line between this line  (``.. code-block:: none``) and the next line (``presto> set... ``) causes the entire code block to not display. I tested this in a local doc build, screenshots: 

without: 
<img width="1020" alt="Screenshot 2024-10-23 at 11 47 36 AM" src="https://github.com/user-attachments/assets/04a6c1e7-fd26-41a0-a933-27fda4b559b2">

and with: 
<img width="1025" alt="Screenshot 2024-10-23 at 11 53 39 AM" src="https://github.com/user-attachments/assets/7bf78870-9c62-4875-ba16-14ecfa9649fc">

Reply:
oops must have deleted that line after i did a local build. Thanks for catching it!

Reply:
No problem! This is why I do a local doc build every time I do a doc review: I could easily have missed this only reading the .rst files in GitHub.